server/Dockerfile

```diff
@@ -54,8 +45,6 @@ ENV JAVA_OPTS '-Dlog4j.configurationFile=file:log4j2.properties'
 
 ENTRYPOINT ["./docker-entrypoint.sh"]
 
-CMD ["dockerize", "-template", "database.properties.tmpl:database.properties", \
-    "-template", "log4j2.properties.tmpl:log4j2.properties", \
-    "./bin/scalardb-server", "--config=database.properties"]
+CMD ["./bin/scalardb-server", "--config=database.properties"]
```

Comment:
We don't need to run the `dockerize` command before running the `scalardb-server` command.